fstat/model.py

```diff
@@ -10,12 +10,25 @@
 )
 
 
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    github_username = db.Column(db.Integer, unique=True)
```

Comment:
Is there a specific reason to call it `github_username` and not just `username`?